src/api/app/models/user.rb

```diff
@@ -862,16 +862,9 @@ def to_param
 
   def nr_of_requests_that_need_work
     Rails.cache.fetch("requests_for_#{login}", expires_in: 2.minutes) do
-      nr_requests_that_need_work = 0
-
-      rel = BsRequestCollection.new(user: login, states: %w(declined), roles: %w(creator))
-      nr_requests_that_need_work += rel.ids.size
-
-      rel = BsRequestCollection.new(user: login, states: %w(new), roles: %w(maintainer))
-      nr_requests_that_need_work += rel.ids.size
-
-      rel = BsRequestCollection.new(user: login, roles: %w(reviewer), reviewstates: %w(new), states: %w(review))
-      nr_requests_that_need_work += rel.ids.size
+      BsRequest.collection(user: login, states: %w(declined), roles: %w(creator)).count +
+      BsRequest.collection(user: login, states: %w(new), roles: %w(maintainer)).count +
+      BsRequest.collection(user: login, roles: %w(reviewer), reviewstates: %w(new), states: %w(review)).count
```

Comment:
Line is too long. [109/80]<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.